src/NuGet.Core/NuGet.Commands/RestoreCommand/Utility/SpecValidationUtility.cs

```diff
@@ -26,6 +27,11 @@ public static void ValidateDependencySpec(DependencyGraphSpec spec)
         /// Validate a dg file. This will throw a RestoreSpecException if there are errors.
         /// </summary>
         public static void ValidateDependencySpec(DependencyGraphSpec spec, HashSet<string> projectsToSkip)
+        {
+            ValidateDependencySpec(spec, projectsToSkip, new NullLogger());
```

Comment:
I believe the singleton could be used here.

```suggestion
            ValidateDependencySpec(spec, projectsToSkip, NullLogger.Instance);
```